vaadin-maps-leaflet-flow-demo/src/main/java/software/xdev/vaadin/maps/leaflet/flow/DemoView.java

```diff
@@ -101,7 +102,64 @@ protected void onAttach(final AttachEvent attachEvent)
 		));
 	}
 	
-	record Example(String route, String name, String desc)
-	{
-	}
+	static final class Example
```

Comment:
I'd add a todo to remember to transform this into a record when possible, to minimize differences with the original.

Reply:
I have created VML 2, that ask to revert all this commit, when we can move to Vaadin24